kotlin-mui-base/src/jsMain/generated/mui/base/useSlider.types.kt

```diff
@@ -156,24 +156,24 @@ external interface UseSliderReturnValue {
 
     /**
      * Resolver for the hidden input slot's props.
-     * @param otherHandlers props for the hidden input slot
+     * @param externalProps props for the hidden input slot
      * @returns props that should be spread on the hidden input slot
      */
-    var getHiddenInputProps: react.Props /* <TOther extends EventHandlers = {}>(otherHandlers?: TOther) => UseSliderHiddenInputProps<TOther> */
+    var getHiddenInputProps: react.Props /* <ExternalProps extends Record<string, unknown> = {}>(externalProps?: ExternalProps) => UseSliderHiddenInputProps<ExternalProps> */
```

Comment:
Strict handler expected in such cases

Reply:
What do you mean? 
Here is only change in comment

Reply:
Strict type expected instead of comment

Reply:
Ok, will do it later in separate task. It is only version update with no changes in this place